docs/src/main/asciidoc/_elytron/migrate/SSL_with_Client_Cert_Migration.adoc

```diff
@@ -164,6 +164,24 @@ Resulting in: -
 </subsystem>
 ----
 
+If you want to use the HTTP DIGEST authentication mechanism in the domain mode behind a load balancer, you can add the option below to the HTTP server mechanism factory:
```

Comment:
What about this logic is specific to "domain mode"?

Reply:
Fixed, thanks